### PR TITLE
feat: Add AutoScaling ScheduledActions

### LINKED ISSUE #615: Add Support AWS::AutoScaling::ScheduledAction
AWS::AutoScaling::ScheduledAction

details can be found [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-as-scheduledaction.html)

### PR DESCRIPTION
**Pull request by spangenberg**
Fixes #615

### CODE REVIEW

File: resources/services/autoscaling/scheduled_actions.go
Comment (roneli) [inline resources/services/autoscaling/scheduled_actions.go:68]: scheduled_action_arn -> arn

File: resources/services/autoscaling/scheduled_actions.go
Comment (roneli) [inline resources/services/autoscaling/scheduled_actions.go:74]: scheduled_action_name -> name

**Review by roneli — COMMENTED** [review]
Two nits, will approve once e2e tests pass.

**Comment by spangenberg** [comment]
Blocked on #703

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**
